records/board/kl27z_microbit_bl.yaml

```diff
@@ -1,4 +1,6 @@
 common:
+    macros:
+        - MICROBIT_LOCK_BOOTLOADER
```

Comment:
FYI, @mbrossard. The MICROBIT_LOCK_BOOTLOADER symbol was on its own "common:" section in the yaml file and somehow was ignored by progen I think. I moved it to the existing "common:" section in the yaml file and now the symbol was seen correctly by the preprocessor.